README.md

```diff
@@ -22,12 +22,12 @@ use:
 <dependency>
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>7.0.0</version>
+  <version>8.0.1</version>
 </dependency>
 ```
 
 ## Installation (Most Up To Date)
-[![Build Status](https://travis-ci.org/ebx/ebx-linkedin-sdk.svg?branch=dev)](https://app.travis-ci.com/github/ebx/ebx-linkedin-sdk)
+[![ebx-linkedin-sdk](https://circleci.com/gh/ebx/ebx-linkedin-sdk.svg?style=svg)](https://app.circleci.com/pipelines/github/ebx/ebx-linkedin-sdk)
```

Comment:
What's this supposed to indicate exactly? I see this, instead of build:passing as in dev:
![Screenshot 2024-05-14 at 10 24 48](https://github.com/ebx/ebx-linkedin-sdk/assets/59833152/8a51da0a-273c-4eb6-be35-e416abd90502)

Reply:
From our discussion in Slack, I think that the code that I have used is correct. However, it will return `FAILED` if reporting the status isn't public. You can see in [this](https://app.circleci.com/pipelines/github/ebx/ebx-linkedin-sdk) link that the status for the dev/master build is shown as `Unauthorized`.